sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerClient.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer;
+
+import com.azure.ai.formrecognizer.models.FormContentType;
+import com.azure.ai.formrecognizer.models.OperationResult;
+import com.azure.ai.formrecognizer.models.ReceiptPageResult;
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.util.IterableStream;
+import com.azure.core.util.polling.SyncPoller;
+import reactor.core.publisher.Flux;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.ByteBuffer;
+
+/**
+ * This class provides a synchronous client that contains all the operations that apply to Azure Form Recognizer.
+ * Operations allowed by the client are, detect language, recognize entities, recognize PII entities,
+ * recognize linked entities, and analyze sentiment for a text input or a list of text inputs.
+ *
+ * <p><strong>Instantiating a synchronous Form Recognizer Client</strong></p>
+ * {@codesnippet com.azure.ai.formrecognizer.FormRecognizerClient.instantiation}
+ *
+ * <p>View {@link FormRecognizerClientBuilder this} for additional ways to construct the client.</p>
+ *
+ * @see FormRecognizerClientBuilder
+ */
+@ServiceClient(builder = FormRecognizerClientBuilder.class)
+public final class FormRecognizerClient {
```

Comment:
Per our conversation today, we will likely split this into multiple clients going forward.  We decided it's ok to diverge for the Preview 1 release.